_includes/releases/v22.1/v22.1.6.md

```diff
@@ -0,0 +1,134 @@
+  - `HashJoinCount`: The number of hash joins in the query plan.
+  - `CrossJoinCount`: The number of cross joins in the query plan.
+  - `IndexJoinCount`: The number of index joins in the query plan.
+  - `LookupJoinCount`: The number of lookup joins in the query plan.
+  - `MergeJoinCount`: The number of merge joins in the query plan.
+  - `InvertedJoinCount`: The number of inverted joins in the query plan.
+  - `ApplyJoinCount`: The number of apply joins in the query plan.
+  - `ZigZagJoinCount`: The number of zig zag joins in the query plan. [#85337][#85337] [#85743][#85743]
+
+<h3 id="v22-1-6-operational-changes">Operational changes</h3>
+
+- Telemetry logs will now display more finely redacted error messages from SQL execution. Previously, the entire error string was fully redacted. [#85403][#85403]
+
+<h3 id="v22-1-6-command-line-changes">Command-line changes</h3>
+
+- The CLI now contains a flag (`--log-config-vars`) that allows for environment variables to be specified for expansion within the logging configuration file. This allows a single logging configuration file to service an array of sinks without further manipulation of the configuration file. [#85171][#85171]
+
+<h3 id="v22-1-6-api-endpoint-changes">API endpoint changes</h3>
+
+- A new `/api/v2/sql/` endpoint enables execution of simple SQL queries over HTTP. [#84374][#84374]
+
+<h3 id="v22-1-6-bug-fixes">Bug fixes</h3>
+
+- Fixed an issue with incorrect start time position of selected time range on the [Metrics page](../v22.1/ui-overview.html#metrics). [#85835][#85835]
+- Fixed an issue where the [`information_schema`](../v22.1/information-schema.html) and [`SHOW GRANTS`](../v22.1/show-grants.html) command did not report that object owners have permission to `GRANT` privileges on that object. [#84918][#84918]
+- Fixed an issue where imports and rebalances were being slowed down due to the accumulation of empty directories from range snapshot applications. [#84223][#84223]
+- The v22.1 upgrade migration `21.2-56: populate RangeAppliedState.RaftAppliedIndexTerm for all ranges` is now more resilient to failures. This migration must be applied across all ranges and replicas in the system, and can fail with `operation "wait for Migrate application" timed out` if any replicas are temporarily unavailable, which is increasingly likely to happen in large clusters with many ranges. Previously, this would restart the migration from the start. [#84909][#84909]
+- Fixed a bug where using `CREATE SCHEDULE` in a mixed version cluster could prevent the scheduled job from actually running because of incorrectly writing a lock file. [#84372][#84372]
+- Previously, backup [restores](../v22.1/backup-and-restore-overview.html) on mixed-version clusters that had not yet finalized the upgrade to v22.1 could fail with `cannot use bulkio.restore_at_current_time.enabled until version MVCCAddSSTable`. Restores now fall back to the v21.2 behavior instead of erroring in this scenario. [#84641][#84641]
```

Comment:
```suggestion
- Previously, [restoring from backups](../v22.1/backup-and-restore-overview.html) on mixed-version clusters that had not yet upgraded to v22.1 could fail with `cannot use bulkio.restore_at_current_time.enabled until version MVCCAddSSTable`. Restores now fall back to the v21.2 behavior instead of erroring in this scenario. [#84641][#84641]
```